src/cli.js

```diff
@@ -36,6 +47,22 @@ function checkInternet() {
 }
 
 if (require.main === module) {
+  const sanitizedArgs = [];
+  process.argv.forEach((arg) => {
```

Comment:
Yeah you're right to do what ESLint asks for.
The weird thing is that CodeClimate report this as an ESLint error.
Dunno what's going on. Disabling the eslint CodeClimate plugin, we have it in the regular `ci` npm task.

Reply:
Okay, I pushed a fix in your branch, CodeClimate and Travis are both happy now. Waiting for your last fix and test.